vegassist.js

```diff
@@ -12,7 +12,15 @@ var DELAYTIME = 1000 * 60 * 2;
 // Load filters from all files in the filters directory
 var settingsFilteredTerms = settings.FILTERED_TERMS || [];
 var settingsFilters = settings.FILTERS || [];
-var filter = new TweetFilter('filters', settingsFilteredTerms, settingsFilters);
+
+var handle = settings.HANDLE || "";
+if (handle.length > 0){
+	if (handle.indexOf('@') != 0)
+		handle = '@' + handle;
+	util.track(handle);
```

Comment:
Ah, right, missed that. I see what's going on now--the handle is being added *as a* filter. Nevermind, then. :+1: